src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -105,72 +105,67 @@ public static bool SequenceEqual(ref byte first, ref byte second, int length)
             int index = 0;
```

Comment:
I must admit I expected a compile error from `fixed (byte* pSearchSpace = &searchSpace)` but it works 😄